server/services/ai.py

```diff
@@ -171,20 +173,60 @@ def _iterate_property_value(
             yield (key, value)
 
 
+def get_places(query: str) -> Sequence[language_v1.Entity]:
```

Comment:
can we start adding documentation for newly added functions?

Reply:
Done.